scripts/dataset/export/index.test.js

```diff
@@ -0,0 +1,148 @@
+import fs from 'fs/promises';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+import chai from 'chai';
+import config from 'config';
+import dircompare from 'dir-compare';
+import mime from 'mime';
+import StreamZip from 'node-stream-zip';
+
+import GitAdapter from '../../../src/storage-adapters/git/index.js';
+
+import generateArchive from './index.js';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+const { expect } = chai;
```

Comment:
> Move these imports before our local imports.

Reply:
By the way, shouldn't eslint take care of sorting the imports itself?